front/src/applications/operationalStudies/hooks/useSetupItineraryForTrainUpdate.ts

```diff
@@ -50,6 +50,17 @@ const useSetupItineraryForTrainUpdate = (
       const trainSchedule = await getTrainScheduleById({
         id: trainIdToEdit,
       }).unwrap();
+      if (!trainSchedule.rolling_stock_name) {
+        adjustConfWithTrainToModifyV2(
```

Comment:
Note: this fixes the edit form not getting populated with the train schedule details in case there is no rolling stock, however there are more error cases below where we should populate the form but don't do it, for instance when pathfinding fails. We should really fix that but it's orthogonal to this PR.